lib/eblox/data/post.ex

```diff
@@ -19,6 +19,40 @@ defmodule Eblox.Data.PostFSM do
 
   use Finitomata, @fsm
 
+  defmodule Tag do
+    @moduledoc false
+    @behaviour Md.Transforms
+
+    @href "/tags/"
+
+    @impl Md.Transforms
+    def apply(md, text) do
+      {:a, %{class: "tag", href: @href <> text}, [md <> text]}
+    end
+  end
+
+  defmodule EbloxParser do
+    @moduledoc false
+    @eblox_syntax Md.Parser.Syntax.merge(
+                    Application.compile_env(:eblox_syntax, :md_syntax, %{
+                      magnet: [
+                        {"🏷️", %{transform: Tag, terminators: [], greedy: false}}
```

Comment:
Extended `Md` syntax.

Reply:
There is an interesting issue with this tag emoji. You added it here with some additional character. The hex of the emoji is `f0 9f 8f b7` [1] but your sequence is `f0 9f 8f b7 ef b8 8f`. I wonder if we should use something more unambiguous to avoid this confusion.

[1] - https://unicode-table.com/en/1F3F7

Reply:
Of course, it’d be also hard to type, I added it as an emoji to also test if `Md` can handle it.

`#` is reserved for the header, `@` is for the mention. Maybe `%`? That would enforce people to use `50\%` for normal percentages, but I think it is an acceptable collateral damage.

Reply:
Or maybe it’d even work out of the box without side effects, I frankly don’t remember how an empty magnet is treated; probably it should be passed through as is, and since nobody types a letter after `%`, we are all set.

Reply:
As far as I know, some parsers treat `#` as a header only if it is followed by a space. So technically speaking it is possible to support both `# header` and `#tag`. For example, Obsidian handles formatting this way: https://help.obsidian.md/How+to/Format+your+notes.

Reply:
It’s still a matter of configuration, which will be later exposed to the user, so let’s now start with the simplest solution. I’m fine with `# ` as a header, and `#` as a tag, or even without `H1` at all, but how would we treat `## ` then? Tag `#` or `H2`?

Let’s now go with `%`, it does not actually matter at all but would be less cumbersome.